packages/soci-snapshotter/soci-snapshotter.service

```diff
@@ -0,0 +1,42 @@
+# Copyright The Soci Snapshotter Authors.
```

Comment:
Isn't this service in the pulled sources? If it is, you can just copy it from them, instead of creating a new service.

Reply:
The upstream has this file but the paths are different:
```
37c37
< ExecStart=/usr/bin/soci-snapshotter-grpc
---
> ExecStart=/usr/local/bin/soci-snapshotter-grpc

```
So close, but no.